tests/vppinputasync.cpp

```diff
@@ -109,4 +109,25 @@ bool VppInputAsync::init(const char* inputFileName, uint32_t fourcc, int width,
 {
     assert(0 && "no need for this");
     return false;
+}
+
+int VppInputAsync::getWidth()
+{
+    if (m_input && m_width != m_input->getWidth())
```

Comment:
thanks @xuguangxin , but  if this function:
`bool VppInputAsync::init(const char* inputFileName, uint32_t fourcc, int width, int height)`
is called, m_input will be null.